R/filter_date_time_range.R

```diff
@@ -0,0 +1,139 @@
+extract_selected_range.date_time_range <- function(filter, range, parent_range, reset) {
```

Comment:
Can we resign from using S3 for this function?
I get warnings related to not exporting them, but I don't think we'd like to export this method.